ash/src/vk/prelude.rs

```diff
@@ -1,3 +1,5 @@
+use crate::vk;
```

Comment:
`ColorComponentFlags` uses `super::`, perhaps we should update that if everyone agrees going through the crate root is cleaner?

Reply:
Addendum: The documentation for `Packed24_8` goes through `super::` as well, I think I did that on purpose.

I bet we can remove those indirections (ie. ``[`vk::AccelerationStructureInstanceKHR`][super::AccelerationStructureInstanceKHR]`` -> ``[`vk::AccelerationStructureInstanceKHR`]``) if `vk` is imported in scope.

Reply:
Ah, I saw a lot of `use crate::vk::submodule::*;` in `ash::vk::constants`/`ash::vk::definitions` etc. I'm also happy to go with `super::Stuff` for consistency with `ColorComponentFlags`.

Reply:
That's how the generator emits those since the old days, I haven't cleaned them up but it's probably good practice to do so. My only reason for preferring this is making the module oblivious of how and where it is located within the `crate`, making it a bit easier to relocate even if that's something we'd never do. I'm sure @Ralith has a better judgement here, let's wait for that.

Reply:
Fwiw this change is being test-driven in #559, documentation is broken anyway :(

Reply:
I favor going through `vk` rather than `super`; it's more concise and more consistent with how everyone's used to dealing with ash types. I'm not necessarily opposed to globbing in modules from the same crate either, but it's probably not justified here.